text/0010-dependency-mappings.md

```diff
@@ -13,16 +13,16 @@ Consumers of Paketo buildpacks may be building in an environment where certain d
 ### Discovery
 If one or more bindings of kind or type `dependency-mapping` are present, then
 a buildpack should use the bindings in its dependency resolution process. The
-dependency's digest, included as `checksum` in the form of `<algorithm>:<hash>`
-(or `sha256` in the form of `<hash>`), should be used to lookup the alternatively
-mapped URI.
+dependency's digest, included as `checksum` in the form of
+`<algorithm>:<hash>` or `<algorithm>-<hash>`, (or `sha256` in the form of
+`<hash>`), should be used to lookup the alternatively mapped URI.
```

Comment:
tl;dr - I'm fine with this approach. 

Question: what about defining a configurable delimiter instead of adding a second delimiter to the RFC? `BP_BINDING_ALGORITHM_DELIMITER` or something like that? It could default to `:` which is the present behavior and then could be changed to any other value that is useful.

Two is probably enough that we won't have to revisit this again, but making it configurable would guarantee we don't have to rehash this topic again. It also follows similar patterns in the Cloud-Native buildpacks spec.

Reply:
We discussed this at the working group meeting this week so I'll summarize my stance - I think I lean towards having the simplest possible user experience.  While you're right, it would be good to avoid having this discussion again in the future by making it configurable, I think that the two options probably will suffice as you said. In general, I'd love for this feature to just work for users without needed to add additional configuration, especially because dependency mappings can already be confusing as is. I'd prefer avoiding any additional user-provided configuration.